pkg/profile/profile.go

```diff
@@ -23,10 +23,11 @@ type ProcessRawData struct {
 }
 
 type RawSample struct {
-	TID         PID
-	UserStack   []uint64
-	KernelStack []uint64
-	Value       uint64
+	TID              PID
+	UserStack        []uint64
+	KernelStack      []uint64
+	InterpreterStack []uint64
```

Comment:
We should comment that the values here are not addresses but instead symbol IDs that need to be looked up in a BPF map.